dietpi/dietpi-software

```diff
@@ -14026,8 +14113,8 @@ _EOF_
 		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == -1 )); then
 
 			Banner_Uninstalling
-			G_AGP steam
-			rm -Rf /{root,home/*}/.steam /mnt/dietpi_userdata/steam
+			G_AGP steam-launcher
```

Comment:
Let's still purge the `steam` package for installs prior to v6.35 or in case it got installed otherwise.